### PR TITLE
Implemented inventory system to pick and equip swords

### PR DESCRIPTION
**Pull request by DanielEmmanuel1**
Summary
- tweaked AtivationWeapon script to mark a weapon as pickable OnTriggerEnter and unmark it OnTriggerExit
- ActivateWeapon function loads up the respective Scriptable Object in the first available slot
- Created 2 Item Scriptable Objects to house the data for two swords in the game
- Created 2 sword prefabs
- Added listeners to Item Slot buttons to equip swords OnClick using UnityEvent 
- Created sprites for the 2 swords (to be displayed in the inventory)
- Sorted the added files into folders

### CODE REVIEW

**Comment by DanielEmmanuel1** [comment]
@sebas11042 @CristopherAguilar10 Please review
All changes done

**Comment by DanielEmmanuel1** [comment]
@CristopherAguilar10 Please review

File: Unity/Assets/Scripts/WeaponsScripts/CollectionWeapons.cs [hunk 1]
```diff
@@ -14,26 +14,45 @@ public class CollectionWeapons : MonoBehaviour
     /// Each element represents a different weapon (sword, axe, etc.) that can be activated.
```
Comment (CristopherAguilar10) [inline Unity/Assets/Scripts/WeaponsScripts/CollectionWeapons.cs:14]: image
Please add the nameSpace line

File: Unity/Assets/Scripts/CharacterSelectionScript/LoadCharacter.cs [hunk 2]
```diff
@@ -10,6 +10,8 @@
 /// </summary>
```
Comment (CristopherAguilar10) [inline Unity/Assets/Scripts/CharacterSelectionScript/LoadCharacter.cs:10]: image
and here add the using

**Review by CristopherAguilar10 — CHANGES_REQUESTED** [review]

File: Unity/Assets/Scripts/CharacterSelectionScript/LoadCharacter.cs (hunk 2 above)
Comment (DanielEmmanuel1) [reply Unity/Assets/Scripts/CharacterSelectionScript/LoadCharacter.cs:10]: Okay, I'll get this sorted

File: Unity/Assets/Scripts/WeaponsScripts/CollectionWeapons.cs (hunk 1 above)
Comment (DanielEmmanuel1) [reply Unity/Assets/Scripts/WeaponsScripts/CollectionWeapons.cs:14]: I'll handle this

**Pull request closed without merging**